catalog-info.yaml

```diff
@@ -12,4 +12,4 @@ metadata:
 spec:
   type: service
   lifecycle: production
-  owner: sre-compute
+  owner: infra-compute-us
```

Comment:
Maybe add a comment with a link to where someone could find a list of valid entries for the "owner" field?

Reply:
The valid owners would be any group entity which exists in [backstage](https://backstage.segmentops.com/catalog?filters%5Bkind%5D=group&filters%5Btags%5D=tda). I will check to make sure our user guide has this rather than adding a comment here as we don't want to update every catalog file with this comment